src/logdata/src/logdata.cpp

```diff
@@ -278,7 +278,7 @@ LineLength LogData::doGetLineLength( LineNumber line ) const
         return 0_length; /* exception? */
     }
 
-    return LineLength( doGetExpandedLineString( line ).size() );
+    return LineLength( static_cast<int>(doGetExpandedLineString( line ).size()) );
```

Comment:
This should be static_cast<LineLength::UnderlyingType>. Qt5 and Qt6 have different limits for the string length. It is not properly supported right now because some places in code do cast to int. I am in the middle of fixing things.